src/c/core/serialization/xrce_protocol_internal.h

```diff
@@ -216,6 +216,7 @@ typedef struct CLIENT_Representation
     uint8_t session_id;
     bool optional_properties;
     PropertySeq properties;
+    uint16_t mtu;
```

Comment:
I don't see the serialization of `CLient_Representation` modified for this new attribute.

Reply:
You are right, sorry. I modified `CLIENT_Representation` de-serialization functions.